include/odp/api/spec/timer.h

```diff
@@ -137,9 +141,29 @@ odp_timer_pool_t odp_timer_pool_create(const char *name, const odp_timer_pool_pa
  * The purpose of this call is to coordinate the creation of multiple timer
  * pools that may use the same underlying HW resources.
  * This function may be called multiple times.
+ *
+ * @deprecated Use odp_timer_pool_start_multi() instead
  */
 void odp_timer_pool_start(void);
 
+/**
+ * Start timer pools
+ *
+ * Start given timer pools. After a pool has been successfully started the pool handle can be used
+ * with other APIs. Each timer pool can be started only once.
+ *
+ * Returns 'num' when all given timer pools have been successfully started. If the return value
+ * N < 'num', only the first N pools started successfully and at least some of the remaining ones
+ * failed to start. The remaining timer pools cannot be used anymore (can only be destroyed).
+ *
+ * @param timer_pool  Array of timer pool handles
+ * @param num         Number of pools to start
+ *
+ * @retval  num on success
+ * @retval <num on failure
```

Comment:
Could mention that negative return value is possible (== 0 pools started)

Reply:
Changed in V3.